Public/Invoke-Locksmith.ps1

```diff
@@ -72,7 +72,16 @@
         [Parameter()]
             [ValidateSet('Auditing','ESC1','ESC2','ESC3','ESC4','ESC5','ESC6','ESC8','All','PromptMe')]
             [array]$Scans = 'All',
-        [string]$OutputPath = (Get-Location).Path,
+        
+        # The directory to save the output in (defaults to the current working directory).
+        [Parameter()]
+            [ValidateScript({(Test-Path -Path $_ -PathType Container)},
+                ErrorMessage = "{0} is not a valid directory path."
```

Comment:
I don't think the ErrorMessage property is supported in Windows PowerShell 5.1. Works well in 7.4 tho

Reply:
Ah, you're right. Will remove.